### PR TITLE
Update tools.md

### PR DESCRIPTION
**Pull request by lumjjb**

### CODE REVIEW

**Review by PushkarJ — CHANGES_REQUESTED** [review]

**Review by achetal01 — APPROVED** [review]

File: governance/tools.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-Documents, such as [self-assessments] are sometimes initially drafted in a Google doc, which facilitates discussion. At the end of the
-review, the documents needs to be converted to markdown so that they can be added to the github repository.
+Documents, such as [self-assessments] are sometimes initially drafted in a
+Google doc, which facilitates discussion. At the end of the review, the
+documents needs to be converted to markdown so that they can be added to the
+github repository.
 
+<!-- cSpell:ignore Pandoc -->
 List of tools:
 - [Pandoc](https://pandoc.org/)
```
[the remark this answers is not in the document]
Comment (lumjjb) [reply governance/tools.md:17]: what's example 35?

[the remark this answers is not in the document]
Comment (lumjjb) [reply governance/tools.md:17]: oh i think i see what you mean, we probably want to be more verbose about it

[the remark this answers is not in the document]
Comment (PushkarJ) [reply governance/tools.md:17]: Updated the suggested commit

File: governance/tools.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
 # Tools
 
-This document contains a set of tools that will be useful to security reviewers and project leads. This is not an endorsement of any vendor or open source project, rather a collection of what has been useful to individuals to help share helpful suggestions to others in the group.
+This document contains a set of tools that will be useful to security reviewers
+and project leads. This is not an endorsement of any vendor or open source
+project, rather a collection of what has been useful to individuals to share 
+helpful suggestions to others in the group.
```
Comment (lumjjb) [inline governance/tools.md:6]:
```suggestion
This document contains a set of tools that will be useful to security reviewers
and project leads. This is not an endorsement of any vendor or open source
project, rather a collection of what has been useful to individuals to share
helpful suggestions to others in the group.
```

File: governance/tools.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+Documents, such as [self-assessments] are sometimes initially drafted in a
+Google doc, which facilitates discussion. At the end of the review, the
+documents needs to be converted to markdown so that they can be added to this
+GitHub repository.
 
+<!-- cSpell:ignore Pandoc -->
 List of tools:
-- [Pandoc](https://pandoc.org/)
-- [Google Docs extension: docs-to-markdown](https://chrome.google.com/webstore/detail/docs-to-markdown/igffnbdfnodiaphfmfaiiaegmoljbghf)
+- [Pandoc](https://pandoc.org/demos.html): Check example in this link to convert `docx` to `markdown`
```
Comment (lumjjb) [inline governance/tools.md:17]:
```suggestion

- [Pandoc](https://pandoc.org/demos.html): Check example in this link to convert
`docx` to `markdown` 
```

File: governance/tools.md [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+documents needs to be converted to markdown so that they can be added to this
+GitHub repository.
 
+<!-- cSpell:ignore Pandoc -->
 List of tools:
-- [Pandoc](https://pandoc.org/)
-- [Google Docs extension: docs-to-markdown](https://chrome.google.com/webstore/detail/docs-to-markdown/igffnbdfnodiaphfmfaiiaegmoljbghf)
+
+- [Pandoc](https://pandoc.org/demos.html): Check example in this link to convert
+`docx` to `markdown`
```
Comment (lumjjb) [inline governance/tools.md:19]:
```suggestion
`docx` to `markdown`
```

**Review by PushkarJ — APPROVED** [review]

**Pull request merged**